src/components/FieldWrapper/__tests__/FieldWrapper.spec.js

```diff
@@ -19,6 +19,20 @@ describe('FieldWrapper', () => {
         expect(toJson(wrapper)).toMatchSnapshot();
         expect(wrapper.find('button')).toHaveLength(1);
     });
+    it('should render arrow icon pointing down', () => {
+        const wrapper = mount(<FieldWrapper showArrow>some children</FieldWrapper>);
+
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+    it('should render arrow icon pointing up', () => {
+        const wrapper = mount(
+            <FieldWrapper showArrow isArrowUp>
+                some children
+            </FieldWrapper>
+        );
+
+        expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
I find  snapshot tests are good for general DOM layout but they can be too easily updated and hence can easily going wrong in the next iteration. Can you add checks to what in snapshot are we looking for?

Reply:
It works! I thought we can find only HTML tags.